### PR TITLE
Cf ab welcome screen

### LINKED ISSUE #8: 7. As a user, I want to see a welcoming prompt to add my first item if my list is empty to help me get oriented to how the app works.
AC:

* The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item

### PR DESCRIPTION
**Pull request by ablizben**
_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

Added logic to the ItemList component that renders a p tag to let the user know when their shopping list is empty and a button to redirect to the AddItems component to allow the user to add an item. 

## Related Issue

Closes #8 

## Acceptance Criteria

- The list view, when there are no items to display, should show a prompt (e.g., a button) for the user to add their first item.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|    | :bug: Bug fix              |
|  ✓ | :sparkles: New feature     |
|    | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

### Before

image

### After

Screen Cast 2021-04-26 at 7 57 18 PM

## Testing Steps / QA Criteria

To test the feature, use one of the following ways:

Follow the link to see the preview: 
Launch the app locally from your terminal: https://tcl-22-shopping-list--pr36-cf-ab-welcome-screen-f7ehja2u.web.app
- Pull down this branch with git pull origin cf-ab-welcome-screen and check that branch out with git checkout origin cf-ab-welcome-screen
- Run npm start to launch the app in your browser at http://localhost:3000

When you first open the app, you should see the welcome message at `/`.

If you either enter a token to join a list or click on create a new list, you will be redirected to `/list`. 

If there are no items in the list, a p tag will render "Your shopping list is currently empty." 

Below the p tag, you can click the "Add Item" button and will be redirected to `/additems`.

### CODE REVIEW

File: src/ItemList.js [hunk 1]
```diff
@@ -1,10 +1,22 @@
 import React from 'react';
+import { useHistory } from 'react-router-dom';
 
 function ItemList(props) {
+  let history = useHistory();
+  const redirect = () => {
+    history.push('/additems');
+  };
```
Comment (dariatsvetkova) [inline src/ItemList.js:8]: Great use of the `useHistory` hook!

**Review by dariatsvetkova — APPROVED** [review]
Nice and clean code. Good job!

**Review by jonathanyeong — APPROVED** [review]
LGTM! Nice use `useHistory`!

**Review by laurenmbeatty — APPROVED** [review]
Looks great!  🎉   😍  Ran Lighthouse to test a11y on this page 
image

**Review by kkatoch89 — APPROVED** [review]
Awesome job! cleanCode && accessible && Killed it 🎉

**Review by luisaugusto — APPROVED** [review]

**Pull request merged**